services/viva/microservice/src/lambdas/submitApplication.ts

```diff
@@ -103,12 +94,12 @@ export async function submitApplication(
   const isRecurringForm = recurringFormId === currentFormId;
   const applicationType = isRecurringForm ? VivaApplicationType.Recurring : VivaApplicationType.New;
   const formId = isRecurringForm ? recurringFormId : newApplicationFormId;
-  const workflowId = details?.workflowId ?? '';
+  const workflowId = details.workflowId ?? '';
   const formAnswers = forms?.[formId].answers ?? [];
   const answers = formAnswers.filter(answer => !dependencies.isAnswerAttachment(answer));
   const attachments = await dependencies.attachmentFromAnswers(personalNumber, formAnswers);
 
-  const [vivaPostError, vivaApplicationResponse] = await to<Record<string, unknown>, VivaPostError>(
+  const [vadaError, vadaResponse] = await to<Record<string, unknown>, VadaError>(
```

Comment:
**Just a note ...**

Sending data like this could possibly be dangerous since lambda has a limit on 6MB size limit on request body payload.

Reply:
Good point
Out of this scoop